lib/samples/style_graphics_with_symbols/README.md

```diff
@@ -0,0 +1,37 @@
+# Style graphics with symbols
+
+Use a symbol style to display a graphic on a graphics overlay.
+
+![image of style graphics with symbols](style_graphics_with_symbols.png)
+
+## Use case
+
+Allows you to customize a graphic by assigning a unique symbol. For example, you may wish to display individual graphics for different landmarks across a region, and to style each one with a unique symbol.  
+
+## How to use the sample
+
+Pan and zoom around the map. Observe the graphics on the map.
+
+## How it works
+
+1. Create a `GraphicsOverlay` and add it to the `MapView`.
```

Comment:
We rename `MapView` to `ArcGISMapView`. The READMEs have to be updated whenever you see one of our renamed classes.